src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1480,6 +1480,17 @@ class ActorIdStringGenerator : public Generator<std::string> {
     std::string _actorId;
 };
 
+/** `{^NowTimestamp: {}}` */
+class NowTimestampGenerator : public Generator<bsoncxx::types::b_timestamp> {
```

Comment:
This could be more generally useful if it could accept a `bsoncxx::types::b_date` from `DateGenerator` (for example) and only default to `now()` if it is not passed in. This would make it properly testable too rather than just checking it doesn't throw

Reply:
Could you please clarify? 
Is you suggestion to use 
`class NowTimestampGenerator : public Generator<bsoncxx::types::b_date>`
instead of 
`class NowTimestampGenerator : public Generator<bsoncxx::types::b_timestamp>` ??